changelog/e2e-7350-shopper-multicurrency-checkout

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: E2E test
```

Comment:
Nit: Should we update this to be something more specific? Maybe, "E2E test - Shopper facing multi-currency checkout"?

Reply:
Done on https://github.com/Automattic/woocommerce-payments/pull/7804/commits/b4e2d9286533a6029e552a93d36044e170b471ce